site/viamguides/src/pi-ai-hat/pi-ai-hat.md

```diff
@@ -0,0 +1,305 @@
+
+Follow the [official documentation](https://www.raspberrypi.com/documentation/accessories/ai-hat-plus.html#ai-hat-plus-installation) for instructions about installing the AI HAT+ on the Raspberry Pi 5.
+
+### What You’ll Need
+
+- All the hardware components listed in prerequisites.
+- Sign up for a free Viam account, and then [sign in](https://app.viam.com/fleet/dashboard) to the Viam app
+
+### What You’ll Learn
+
+- How to configure and test a device's components using Viam
+- How to use modules from the Viam registry
+- How to use a vision service for object detection
+- How to integrate vision service data into a teleop dashboard
+
+### Watch the Video
+
+See a demonstration of the traffic monitor in this video.
+
+**TBD**
+
+<!-- <video id="DH3kkFjmGss"></video> -->
+
+<!-- ------------------------ -->
+
+## Set up your Raspberry Pi
+
+Duration: 5
+
+The Raspberry Pi boots from a microSD card. You need to install Raspberry Pi OS on the microSD card that you will use with your Pi. For more details about alternative methods of setting up your Raspberry Pi, refer to the [Viam docs](https://docs.viam.com/installation/prepare/rpi-setup/#install-raspberry-pi-os).
+
+### Install Raspberry Pi OS
+
+1. Connect the microSD card to your computer.
+1. Download the [Raspberry Pi Imager](https://www.raspberrypi.com/software/) and launch it.
+   ![raspberry pi imager](assets/imager.png)
+1. Click **CHOOSE DEVICE**. Select your model of Pi, which is Raspberry Pi 5.
+1. Click **CHOOSE OS**. Select **Raspberry Pi OS (64-bit)** from the menu.
+1. Click **CHOOSE STORAGE**. From the list of devices, select the microSD card you intend to use in your Raspberry Pi.
+   ![raspberry pi storage](assets/osSelect.png)
+1. Configure your Raspberry Pi for remote access. Click **Next**. When prompted to apply OS customization settings, select **EDIT SETTINGS**.
+1. Check **Set hostname** and enter the name you would like to access the Pi by in that field, for example, `test`.
+1. Select the checkbox for **Set username and password** and set a username (for example, your first name) that you will use to log into the Pi. If you skip this step, the default username will be `pi` (not recommended for security reasons). And specify a password.
+1. Connect your Pi to Wi-Fi so that you can run `viam-server` wirelessly. Check **Configure wireless LAN** and enter your wireless network credentials. SSID (short for Service Set Identifier) is your Wi-Fi network name, and password is the network password. Change the section `Wireless LAN country` to where your router is currently being operated.
+   ![raspberry pi hostname username and password](assets/generalSettings.png)
+1. Select the **SERVICES** tab, check **Enable SSH**, and select **Use password authentication**.
+   ![raspberry pi enable SSH](assets/sshSettings.png)
+   > aside negative
```

Comment:
`aside negative` renders the actual text "aside negative" in the rich diff. Not sure if that goes away after some build process, but wanted to point it out.

Reply:
Yes, the "negative" aside renders this banner UI in the rendered codelab:

<img width="773" alt="image" src="https://github.com/user-attachments/assets/22ce2394-d3c3-4f61-899e-362133b4f533" />

Reply:
Gotcha. Thanks for clarifying again :)